app/assets/javascripts/controllers/CommonChatController.ts

```diff
@@ -552,7 +553,7 @@ export default class CommonChatController {
         } else {
             sessionStorage.removeItem("surveySkipped");
             document.cookie = "surveyed=true";
-            this._sendPostChatSurveyDigitalAssistant(this.sdk).closePostChatSurvey(automatonWebchat, timestamp);
+            this._sendPostChatSurveyWebchat(this.sdk).closePostChatSurvey(automatonWebchat, timestamp);
             this.showEndChatPage(false);
             surveyPage.detach();
         }
```

Comment:
The wrong function seemed to be called here. It was using this._sendPostChatSurveyDigitalAssistant() within the onPostChatSurveyWebchatSubmitted() method.